crates/nu-command/src/strings/str_/replace.rs

```diff
@@ -156,12 +156,17 @@ impl Command for SubCommand {
                 result: Some(Value::test_string("my_library.nu")),
             },
             Example {
-                description: "Find and replace all occurrences of find string using regular expression",
+                description: "Find and replace contents with capture group using regular expression, with escapes",
+                example: "'hello=world' | str replace -ra '\\$?(?<varname>.*)=(?<value>.*)' '$$$varname = $value'",
```

Comment:
doesn't the `a` in `-ra` also count for "all occurrences"? otherwise it may not be needed. 

also, i get an error with the example.
```nushell
Error: nu::shell::incorrect_value

  × Incorrect value.
   ╭─[entry #42:1:17]
 1 │ 'hello=world' | str replace -ra '\\$?(?<varname>.*)=(?<value>.*)' '$$$varname = $value'
   ·                 ─────┬─────     ────────────────┬────────────────
   ·                      │                          ╰── Regex error: Parsing error at position 3: Target of repeat operator is invalid
   ·                      ╰── encountered here
   ╰────
```

Reply:
Make sure you unescape the \\ to \ if you copy paste from the code to the terminal.
If not, build the source, ran the help manually, and there you can copy paste from the terminal to the terminal

Reply:
ah, forgot to unescape. thanks for the clarification.